docs/docs/cloud/how-tos/copy_threads.md

```diff
@@ -0,0 +1,165 @@
+# Copying Threads
+
+You may wish to copy an existing thread in order to keep the existing history and experiment with different runs in the future.
+
+## Setup
+
+To copy a thread, we must first set up our client and also create a thread we wish to copy. 
+
+### SDK initialization
+
+First, we need to setup our client so that we can communicate with our hosted graph:
+
+=== "Python"
+
+    ```python
+    from langgraph_sdk import get_client
+    client = get_client(url="<DEPLOYMENT_URL>")
+    assistant_id = "agent"
+    thread = await client.threads.create()
+    ```
+
+=== "Javascript"
+
+    ```js
+    import { Client } from "@langchain/langgraph-sdk";
+
+    const client = new Client({ apiUrl:"<DEPLOYMENT_URL>" });
+    const assistantId = agent;
+    const thread = await client.threads.create();
+    ```
+
+=== "CURL"
+
+    ```bash
+    curl --request POST \
+      --url <DEPLOYMENT_URL>/threads \
+      --header 'Content-Type: application/json' \
+      --data '{
+        "metadata": {}
+      }'
+    ```
+
+### Thread creation
```

Comment:
Don't think we need this section at all, we can just add a note to the section below saying "This assumes that a thread you'd like to copy already exists"